libs/ebpfnetsh/programs.cpp

```diff
@@ -368,7 +368,8 @@ handle_ebpf_delete_program(
 }
 
 _Must_inspect_result_ ebpf_result_t
-_ebpf_program_attach_by_id(ebpf_id_t program_id, ebpf_attach_type_t attach_type, _In_opt_ LPWSTR interface_parameter)
+_ebpf_program_attach_by_id(
+    ebpf_id_t program_id, ebpf_attach_type_t attach_type, _In_opt_ const LPWSTR interface_parameter)
```

Comment:
I think you can just do
```suggestion
    ebpf_id_t program_id, ebpf_attach_type_t attach_type, _In_opt_ LPCWSTR interface_parameter)
```

Reply:
True, but eventually I was going to replace the Windows API style type definitions with type definitions that match the coding style guidelines. So, LPCWSTR would be replaces with:
```
const _Null_terminated_ wchar_t*
```
At which point I was going to get rid of the double const.

For now adding the const makes it easier to find cases where parameters are marked as _In_ with out const.

Reply:
Filed #1750 to track replacing types that don't conform to the coding guidelines with ones that do.